### ISSUE TITLE
Performance issue

### ISSUE THREAD

**Issue by ws-garcia** [sdkn104/VBA-CSV#2]
I currently work on a CSV-VBA interface that I suposses'll to allow users exchange data between VBA arrays and CSV files, like your project do, but at really high speed. You claim your code is able to read CSV data at about 5.33 MB/sec., but in my test, *_on a fairly outdated laptop, the performance of it is frustrating_*. Some suggestion?

**Comment by sdkn104** [comment]
Thank you for informing your benchmark results.
I have checked your benchmark data and found performance issue in my VBA code.
I have fixed the issue and released in v1.9.  
(InStr() for long string consumed large cpu time in my code for such CSV that includes no double-quote character)
My benchmark on your 100K rec data is now:
    ImportFromCSV: 0.6 sec.
    ParseCSVToArray: 1.9 sec.
on my PC (Core i5-3470 CPU @ 3.2GHz, 4GB memory).
Thank you.
